src/cli.rs

```diff
@@ -527,22 +696,87 @@ pub fn handle_cli_asset_command(command: &AssetCommands) {
                 }
             }
         }
+        AssetCommands::BearerAuth { uuid, token } => {
```

Comment:
New asset command?

Reply:
hmm no, rebase issue I think... need to merge master to this branch

Reply:
fixed